openfe/tests/data/serialization/network_template.graphml

```diff
@@ -0,0 +1,76 @@
+<graphml xmlns="http://graphml.graphdrawing.org/xmlns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://graphml.graphdrawing.org/xmlns http://graphml.graphdrawing.org/xmlns/1.0/graphml.xsd">
+  <key id="d2" for="edge" attr.name="mapping" attr.type="string" />
+  <key id="d1" for="node" attr.name="sdf" attr.type="string" />
+  <key id="d0" for="graph" attr.name="ofe_version" attr.type="string" />
+  <graph edgedefault="directed">
+    <node id="mol0">
+      <data key="d1">
+     RDKit          2D
+
+  2  1  0  0  0  0  0  0  0  0999 V2000
+    0.0000    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    1.2990    0.7500    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0
+M  END
+
+&gt;  &lt;ofe-name&gt;
+
+
+&gt;  &lt;ofe-version&gt;
```

Comment:
it looks like these tags got mangled? or is this correct?

Reply:
This is correct. SDF isn't properly XML (note the lack of close-tags), but it does use XML-like `<tag>` notation. So you can't embed SDF with angle brackets as in the original usual because it isn't XML. It has to be embedded as text, and the correct way to escape angle brackets in XML is through entities.

One of many reasons I do not like XML. It's a shame that the most widely-used graph serialization formats are both XML, but here we are.